website/data/users.json

```diff
@@ -151,5 +151,19 @@
         "infoLink": "https://www.flextrade.com/",
         "pinned": true,
         "isMember": true
+    },
+    {
```

Comment:
Not alpha sorted - seems other recent additions weren't either, needs fixing.

Reply:
...either by alpha sorting the data in the file or before it is rendered here:
https://github.com/finos/FDC3/blob/c2470b2f49eff4cbecf6d436af684a21c67b5874/website/pages/en/users.js#L25-L34

and here:
https://github.com/finos/FDC3/blob/c2470b2f49eff4cbecf6d436af684a21c67b5874/website/pages/en/index.js#L165

Reply:
done now I think